VAMobile/src/components/FormWrapper/FormFields/FormAttachments.test.tsx

```diff
@@ -51,55 +42,45 @@ context('FormAttachments', () => {
     mockNavigateToSpy = jest.fn()
     mockNavigationSpy.mockReturnValue(mockNavigateToSpy)
 
-    component = render(
+    render(
       <FormAttachments
         removeOnPress={removeOnPressSpy}
         largeButtonProps={{ label: 'add files', onPress: largeButtonSpy }}
         attachmentsList={attachments}
       />,
     )
-
-    testInstance = component.UNSAFE_root
   }
 
   beforeEach(() => {
     initializeTestInstance()
   })
 
-  it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
-  })
-
   describe('when there are attachments', () => {
-    it('should display a remove link', async () => {
-      expect(testInstance.findAllByType(TextView)[2].props.children).toEqual('Remove')
+    it('should display a remove link', () => {
+      expect(screen.getAllByText('Remove').length).toBeGreaterThan(0)
     })
 
     describe('when the remove link is clicked for an attachment', () => {
-      it('should call the removeOnPress', async () => {
-        await waitFor(() => {
-          testInstance.findAllByProps({ accessibilityLabel: 'Remove' })[0].props.onPress()
-          expect(removeOnPressSpy).toHaveBeenCalled()
-        })
+      it('should call the removeOnPress', () => {
+        fireEvent.press(screen.getAllByText('Remove')[0])
```

Comment:
should be able to update this to getByRole('link', {name: 'Remove'})

Reply:
Same as above, it has to use the "all" modifier because there are more than one. Updated.